p8e-api/src/main/kotlin/io/provenance/engine/config/Properties.kt

```diff
@@ -135,3 +135,13 @@ class MetricsProperties {
     var tags: String = ""
     var prefix: String = "p8e_api"
 }
+
+@ConfigurationProperties(prefix = "smartkey")
+class SmartKeyProperties {
+    @NotNull lateinit var apiKey: String
```

Comment:
Considered adding a boolean flag for enable/disable, but didn't make sense if we set a flag for each of the signer then developer would expect that it would write to both places, which we want to avoid.